torntools/scripts/content/global/ttGlobal.js

```diff
@@ -148,9 +148,9 @@ requireDatabase().then(() => {
 				time.setAttribute("seconds-up", seconds);
 			}
 		}, 1000);
-		
-		if (userdata.faction.faction_id && settings.pages.global.highlight_chain_timer && settings.pages.global.highlight_chain_length >= 10) chainTimerHighlight();
-		
+
+		if (userdata.faction.faction_id && settings.pages.global.highlight_chain_timer && settings.pages.global.highlight_chain_length >= 10)
+			chainTimerHighlight();
```

Comment:
Really ?
Isn't one line enough there ?

Reply:
I ran Prettier for the formatting, so yes, one line isn't enough.

Reply:
I think that should be changed in Prettier Settings as one line is actually enough there.

Reply:
Prettier recommend to not have `printWidth` over 80. We have it at 160, pretty sure that when Prettier cuts it off, it's too long xD